src/shared/lib/getCanonicalMutationType.ts

```diff
@@ -98,6 +98,7 @@ export function getProteinImpactTypeFromCanonical(mutationType:CanonicalMutation
         case "fusion":
         case "silent":
```

Comment:
@adamabeshouse seems like this code should just have case for the main named types and that have all over cases handled by default.  
(Incidentally, there's a linting rule that requires a default for all switch statements. Would've helped here.)
I'll write a test for this after we decide whether to change code in way I suggested above.